dhis2-android-app/src/main/java/org/dhis2/mobile/processors/ReportUploadProcessor.java

```diff
@@ -85,6 +85,7 @@ public static void upload(Context context, DatasetInfoHolder info, ArrayList<Gro
                     SyncLogger.getResponseDescription(context,response),
                     SyncLogger.getNotification(info));
         } else {
+            DialogHandler.showMessage(SyncLogger.getErrorMessage(context, info, response, true));
```

Comment:
According to the issue description "Show sync errors as pop-ups instead of Android notifications" and you here when an error occurred send notification and show a dialog, we should not send notification only show a dialog.